src/coreclr/nativeaot/Runtime/startup.cpp

```diff
@@ -1,6 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 #include "common.h"
+#ifdef HOST_WINDOWS
+#include <windows.h>
```

Comment:
The PAL just gets into way in the complicated interactions that this requires, putting the code under ifdef is a lot simpler.